keyboards/kinesis/kintwin/rules.mk

```diff
@@ -0,0 +1,15 @@
+DEFAULT_FOLDER = kinesis/kintwin
+
+# Build Options
+#   change yes to no to disable
+#
+BOOTMAGIC_ENABLE = yes     # Enable Bootmagic Lite
+MOUSEKEY_ENABLE = no       # Mouse keys
+EXTRAKEY_ENABLE = no       # Audio control and System control
+CONSOLE_ENABLE = no        # Console for debug
+COMMAND_ENABLE = no        # Commands for debug and configuration
+NKRO_ENABLE = yes          # Enable N-Key Rollover
+BACKLIGHT_ENABLE = no      # Enable keyboard backlight functionality
+RGBLIGHT_ENABLE = no       # Enable keyboard RGB underglow
+AUDIO_ENABLE = no          # Audio output
+KEYBOARD_SHARED_EP = yes   # Combine base keyboard functionality into a single USB endpoint
```

Comment:
I will revert this change for now.

Reply:
Would the build warnings be indicative that each controller/board under `kinesis` dir should have their own rules.mk/info.json defining what feature are enabled/disabled, rather than having a blanket/master rules.mk?

Reply:
Linting check is failing due to no licence headers in identified keymaps.

Reply:
> Would the build warnings be indicative that each controller/board under `kinesis` dir should have their own rules.mk/info.json defining what feature are enabled/disabled, rather than having a blanket/master rules.mk?

Yes, I think master rules.mk is unnecessary. But its removal might result in build option changes for some controllers.

Reply:
> 

What files are missing license headers?

Reply:
> > 
> 
> What files are missing license headers?

`kinesis/keymaps/default/keymap.c` and `kinesis/keymaps/default_pretty/keymap.c`

Reply:
> Yes, I think master rules.mk is unnecessary. But its removal might result in build option changes for some controllers.

Having had a look at `kinesis/rules mk`, thankfully it doesn't contain definitions nor dependencies that would break controllers.

Reply:
Committed requested changes

Reply:
> Committed requested changes

excellent! I'm not sure why change in the most recent commit aven't been reflected in the CI Build, though recently Pull Requests in GitHub (not just qmk_firmware repo) haven't been functioning correctly.
By chance, have you built this firmware locally/on your computer to see if build errors/warning occur?